amicleaner/fetch.py

```diff
@@ -78,6 +107,39 @@ def fetch_zeroed_asg(self):
 
         return amis
 
+    def fetch_zeroed_asg_lt(self):
+
+        """
+        Find AMIs for autoscaling groups who's desired capacity is set to 0
+        """
+
+        resp = self.asg.describe_auto_scaling_groups()
+        # This does not support multiple versions of the same launch template being used
+        zeroed_lts = [asg.get("LaunchTemplate", {})
+                      for asg in resp.get("AutoScalingGroups", [])
+                      if asg.get("DesiredCapacity", 0) == 0]
+
+        zeroed_lt_names = [lt.get("LaunchTemplateName", "")
```

Comment:
This line has same problem with previous error.
The element of `zeroed_lt_names` can be empty string....